src/state/apis/snapshot/snapshot.ts

```diff
@@ -142,5 +147,62 @@ export const snapshotApi = createApi({
         }
       },
     }),
+    getProposals: build.query<Proposal[], void>({
+      queryFn: async (_, { dispatch }) => {
+        const query = `
+          query Proposals {
+            proposals(
+              first: 20,
```

Comment:
suggestion: fetching 20 is a magic heuristics and will only work by virtue of us (realistically) never having more than 15 proposals at the same time but conceptually could be improved without heuristics.

Can we use a gql query like this one?

```gql
query Proposals {
  activeProposals: proposals(
    first: 5,
    where: {
      space: "aave.eth",
      state: "active"
    },
    orderBy: "created",
    orderDirection: desc
  ) {
	id
	title
	body
	choices
	scores
	scores_total
	link
  }
  
  closedProposals: proposals(
    first: 5,
    where: {
      space: "aave.eth",
      state: "closed"
    },
    orderBy: "created",
    orderDirection: desc
  ) {
	id
	title
	body
	choices
	scores
	scores_total
	link
  }
}
```
(note aave.eth as a space, since it has both active and closed proposals, don't forget to use programmatic space back if implementing me)

Changes:

- Only fetches required fields
- Fetches 5 active and 5 closed (not sure how many we want for active, feel free to change, this ensures we always have the number of active/closed we want, and never end up in a situation where we don't fetch enough active/closed because the other side has more than 15)
- We can then remove this guy:

https://github.com/shapeshift/web/blob/386fc66748ac8b1bffaec45838fd0313d2981185/src/pages/Fox/components/FoxGovernance.tsx#L69

Reply:
Done but kept 20 active proposals so the buffer is enough for any active proposals, in [2b38bd6](https://github.com/shapeshift/web/pull/7952/commits/2b38bd691a2b53a6c5fc056cd4fe189082ca3806)